catalog/USGS/NLCD_RELEASES/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_COVER.jsonnet

```diff
@@ -26,44 +26,55 @@ local self_url = catalog_subdir_url + base_filename;
   version: 'v5',
   'gee:type': ee_const.gee_type.image_collection,
   description: |||
-    Rangeland ecosystems in the western United States have a dynamic response
-    to climate change, fire, and other anthropogenic disturbances. The Rangeland
-    Condition, Monitoring, Assessment, and Projection (RCMAP) product aims to
-    capture this response by quantifying the percent cover of rangeland
-    components, associated error, and trends across the western U.S. using
-    Landsat imagery from 1985-2020.
-
-    RCMAP quantifies the percent cover of components across the western U.S.
-    rangelands (after Rigge et al. 2020) using Landsat imagery from 1985-2020.
-    The RCMAP timeseries consists of eight fractional components: annual
-    herbaceous, bare ground, herbaceous, litter, non-sagebrush shrub, perennial
-    herbaceous, sagebrush and shrub, and the temporal trends of each. The four
-    primary components (bare ground, shrub, litter, and herbaceous) are designed
-    to sum to 100% in each pixel when added to tree canopy cover. The secondary
-    components annual herbaceous and perennial herbaceous are subsets of the
-    primary component herbaceous, while non-sagebrush shrub and sagebrush are
-    subsets of shrub. Secondary components cannot have cover greater than their
-    respective primary component. One year, 2012, was excluded from
-    the timeseries due to a lack of quality imagery.
-
-    MRLC developed an automated method to identify change in spectral
-    conditions between each year in the Landsat archive and the circa 2016 base
-    map. Regression tree models were trained from the unchanged portions of each
-    year in the time series. Post-processing models corrected post-burn
-    trajectories and eliminated noise and illogical change in the predictions.
-    The current generation of RCMAP has been improved with more training data,
-    regional-scale Landsat composites, and more robust change detection. MRLC
-    assessed the temporal patterns in each component with a linear model and
-    structural change method which determines break points in the timeseries
-    using an 8-year temporal moving window. The linear and structural change
-    methods generally agreed on gross patterns of change, but the latter found
-    breaks more often with most pixels having at least one break point. Data
-    provide spatiotemporal information on the occurrence of breaks, but even
-    more critically, attribute those change events to specific component(s).
-    The spatially, temporally, and thematically (i.e., multi-component) detailed
-    specific information on rangeland condition can contribute to understanding
-    major patterns of change at local, regional, and continental levels.
-
+      The RCMAP (Rangeland Condition Monitoring Assessment and Projection) dataset quantifies 
```

Comment:
Only indent 2 more from the prior line.  All text lines start at the same indent.

From the CI check run:

```
ERROR: /home/runner/work/earthengine-catalog/earthengine-catalog/catalog/BUILD:6:17: Compiling Jsonnet to JSON for jsonnet_to_json_USGS_NLCD_RELEASES_USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_COVER.jsonnet failed: (Exit 1): bash failed: error executing command /bin/bash -c ... (remaining 1 argument skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
catalog/USGS/NLCD_RELEASES/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_COVER.jsonnet:[28](https://github.com/google/earthengine-catalog/actions/runs/4128241463/jobs/7132436537#step:6:29):16 Text block not terminated with |||

  description: |||
```

Reply:
Fixed